qemu/tests/slof_device_tree.py

```diff
@@ -58,9 +59,10 @@ def check_nonexist_aliases(vm_session):
         uuid = vm.get_uuid()
         compare_dev_tree("system-id", uuid)
         compare_dev_tree("vm,uuid", uuid)
-        host_system_id = process.system_output(
-            "echo `cat /proc/device-tree/system-id`", shell=True).strip().decode()
-        compare_dev_tree("host-serial", host_system_id)
+        if int(utils_misc.get_qemu_version()['major']) < 4:
+            host_system_id = process.system_output(
+                "echo `cat /proc/device-tree/system-id`", shell=True).strip().decode()
```

Comment:
```suggestion
                "cat /proc/device-tree/system-id", shell=True).decode().strip("\x00")
```
Because echo \`cat\` these files will report a warning message
```
-bash: warning: command substitution: ignored null byte in input
```